sdk/src/test/scala/com/horizen/account/api/rpc/service/EthServiceTest.scala

```diff
@@ -37,9 +39,8 @@ class EthServiceTest extends JUnitSuite
     val settings = mock[SidechainSettings]
     params = RegTestParams(initialCumulativeCommTreeHash = FieldElementFixture.generateFieldElement())
     transactionActorRef = mock[ActorRef]
-    val nodeView = mock[CurrentView[AccountHistory, AccountState, AccountWallet, AccountMemoryPool]]
-    val stateView = mock[AccountStateView]
-    ethService = new EthService(stateView, nodeView, params, settings, transactionActorRef)
+
+    ethService = new EthService(transactionActorRef, new FiniteDuration(10, SECONDS), params, settings, transactionActorRef)
```

Comment:
'transactionActorRef' used twice.
I think, we need to declare `NodeViewHolderRef = mock[ActorRef]`
later this mocked object will be used for testing all the rpc methods.